lib/benchmark_harness.dart

```diff
@@ -1,4 +1,4 @@
 library benchmark_harness;
 
-part 'src/benchmark_base.dart';
-part 'src/score_emitter.dart';
+export 'src/benchmark_base.dart' show BenchmarkBase;
```

Comment:
The "show" clauses are not needed.  All the classes from these libraries are exported in the API, so just leave out the show clause.